### ISSUE TITLE
Unable to see performance plots

### ISSUE THREAD

**Issue by psteinb** [sbi-benchmark/streamlit#2]
Hi,

I wanted to share the results from the streamlit app with a collaborator. But I had to discover that the performance plots are not present:
![image](https://github.com/user-attachments/assets/df4b1a04-a725-43d4-94de-c2d139c37915)

Just wanted to know if other people make this observation too ... then it could be my browser setup (ff 130.0).

**Comment by jan-matthis** [comment]
I can repro the issue with posterior samples, along the lines of your screenshot. Metrics/performance plots work fine on FF 131 and Chrome 129 for me.

**Comment by psteinb** [comment]
Yeah, ff 131 apparently "fixed" the issue. Not sure what to do about it. Feels like an interplay between streamlit and the browsers. Feel free to close the issue at will.
